client/reader/conversations/index.js

```diff
@@ -28,10 +31,12 @@ export default function() {
 			updateLastRoute,
 			initAbTests,
 			sidebar,
-			conversationsA8c
+			conversationsA8c,
+			makeLayout,
+			clientRender
 		);
 	} else {
-		page( '/read/conversations', '/' );
-		page( '/read/conversations/a8c', '/' );
+		page( '/read/conversations', '/', makeLayout, clientRender );
+		page( '/read/conversations/a8c', '/', makeLayout, clientRender );
```

Comment:
`page('/foo', '/')` just redirects to `/` I assume? There seems to be quite a few of these as well these:
```js
page( '/payment-methods/add-credit-card', () => page.redirect( paths.addCreditCard() ) );
```

Reply:
> `page('/foo', '/')` just redirects to `/` I assume? (Edit: yep)

I guess we can exclude those by checking for argument number (2) and type of second arg (string literal -- anything else, such as a variable, would probably be an edge case).

Reply:
> There seems to be quite a few of these as well these:
```js
page( '/payment-methods/add-credit-card', () => page.redirect( paths.addCreditCard() ) );
```

Hmm, I might've written some of those. We should probably replace them by `https://github.com/visionmedia/page.js#pageredirectfrompath-topath` which is the most semantic of the lot I guess. Not sure it falls under the scope of the STR project, as long as the codemod manages to deal with this type of route def correctly...

Reply:
>Hmm, I might've written some of those. We should probably replace them by https://github.com/visionmedia/page.js#pageredirectfrompath-topath which is the most semantic of the lot I guess. Not sure it falls under the scope of the STR project, as long as the codemod manages to deal with this type of route def correctly...

Good point. They're in varying formats (sometimes they have logic), sometimes they're just `() => page.redirect( '/foo' )` so it made sense to modify codemod to take them into account:

https://github.com/Automattic/wp-calypso/pull/19859